demo/demo.js

```diff
@@ -185,28 +116,39 @@ function setCookie(cookie, value, isTransient) {
 	const expires = isTransient
 		? new Date(Date.now() + ONE_DAY)
 		: new Date('9999-01-01')
-	document.cookie = `revsdk-${cookie}=${encodeURIComponent(value)};expires=${expires.toUTCString()}`;
+	document.cookie = `${cookie}=${encodeURIComponent(value)};expires=${expires.toUTCString()}`;
 }
 
 function getCookie(name) {
 	let value = '; ' + document.cookie;
-	let parts = value.split(`; refsdk-${name}=`);
+	let parts = value.split(`; ${name}=`);
 	if (parts.length === 2) {
 		return decodeURIComponent(parts.pop().split(';').shift());
 	}
 }
 
-function getQueryParams() {
-	return Object.fromEntries(
-		new URLSearchParams(window.location.search)
-	);
-}
-
 function getParameterByName(name) {
 	return queryParams[name] || getCookie(name) || '';
 }
 
-function writeForm(values) {
+function readParams(defaults) {
+	return Object.keys(defaults).reduce((acc, k) => {
+		acc[k] = getParameterByName(k) || defaults[k];
+		return acc;
+	}, {});
+}
+
+function readFormData(form) {
+	return Array.from(form.elements).reduce((acc, el) => {
+		if(el.name) {
```

Comment:
this won't work for radio elements (all are returned regardless of if they're checked)
logic might be something like `if (el.name && (el.type !== 'radio' || el.checked))`